.travis/install_from_source

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
```

Comment:
minor - recommend 'eux' for set flags. 'u' is just good to have, but no practical benefit here. 'x'  echo's the command being executed before executing it. I'm thinking that may make for good travis output.